src/lib/Navbar.svelte

```diff
@@ -180,6 +180,10 @@
 		header.elevated {
 			box-shadow: none;
 		}
+
+		ul:not(:has(.brand)) {
```

Comment:
Since we're selecting two ul's here and the second one already has a class ("menu"), should we add a class to the first one and select them both by their class instead for readability? Or do you think that's not helpful?

Reply:
Well it depends on how you think about what's happening here. In my mind I'm thinking "only the logo and site name should be visible" so this selector makes more sense to me but if I was thinking "get rid of the navigation links and the menu icon" then I would probably do what you're suggesting. Another thing I ask myself is "if we added a 4th ul to the nav should it also be hidden in print mode?" I think the answer is yes because typically things in the nav are interactive.

Reply:
Yeah that makes alot more sense, thanks. Hypothetically, if we added another ul and were using classes, we'd have to remember to add a class to the new ul, which makes less sense than having a good selector.